pheps/phep-0001.md

```diff
@@ -0,0 +1,287 @@
+
+PHEP authors should incorporate community feedback by making edits and additional pushes to their branch. They must add a new `Post-History` entry added with each push. It is recommended to address as much actionable feedback as practicable with each push; however, each push may certainly contain multiple commits for ease of review. PHEP authors must not rewrite history after a push, i.e. no force-pushing. Rebasing of history before a push for greater clarity is acceptable.
+
+PHEP authors are also responsible for managing the progress of the discussion of their PHEP, for instance by updating the PR description to highlight topics of particular interest and to provide the dates of votes on the PHEP resolution.
+
+If a PHEP undergoes a significant re-write or other major, substantive changes to its proposed specification, the author(s) and editor may discuss closing the existing PR and opening a new one, starting from the changed version. If this occurs, the `Discussions-To` link must be updated.
+
+PHEP authors are responsible for collecting community feedback on a PHEP before submitting it for review. However, to avoid long-winded and open-ended discussions, strategies such as soliciting private or more narrowly-tailored feedback in the early design phase, collaborating with other community members with expertise in the PHEP's subject matter, and picking an appropriately-specialized discussion for the PHEP's topic (if applicable) should be considered. PHEP authors should use their discretion here.
+
+PHEP discussions are subject to the [PyHC Code of Conduct](https://heliopython.org/docs/code_of_conduct/).
+
+## PHEP Review & Resolution
+Once the authors have completed a PHEP, they may request a review for style and consistency from the PHEP editors. However, content review and acceptance of the PHEP is ultimately the responsibility of the community once the authors determine the PHEP is ready for final review and resolution. A PHEP is accepted when approved by consensus of attendees (virtual or in-person) at both a PyHC regular telecon and a regular PyHC meeting with no edits to the PHEP occurring in between. Any edits "reset the clock" and two further votes must be taken. Potential approval of a PHEP must be placed on the agenda at least a month before the meeting, including a link to the open PR with the proposed-final language of the PHEP. Persons not in attendance at the meetings may object via email or Element/Slack chat, provided that they ensure the PHEP author is notified.
```

Comment:
Re: `at least a month before the meeting,` I agree that a month is a good timeline. I'm not sure how enforceable it'll be though; this feels like a detail that would easily get lost in the weeds (e.g. people might remember it should be on the agenda, but might forget the exactly one month part).

Reply:
I'm okay with "oops we need to wait another month". It does seem that the talks are scheduled pretty well a month out, and the meeting _agenda_ is set even if the _schedule_ isn't...as written I think "we will discuss this at the meeting" is sufficient and it doesn't require "we will discuss Monday at 10am", but if anybody is interpreting this differently, we should edit.

@jibarnum , thoughts?

Reply:
Cool. I don't see a real need to update the language here.